internal/versions/rest_api_versions_test.go

```diff
@@ -0,0 +1,786 @@
+package versions
```

Comment:
Can you ensure you retain the tests added in https://github.com/openshift/assisted-service/pull/6066/files#diff-83dbd9742dbbf03982097164da597f3c30dcda4ed0af066409af7f3ac7f67716R281-R330 to ensure we don't regress?

Reply:
Yes I added in kubeAPI mode, in rest we don't use that code